app/models/salt_event_handlers/minion_start.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# This class is responsible to handle the salt events with tag "minion_start".
+# When such an event occurs, we want the minion to be saved in our database
+# if not already there.
+class SaltEventHandlers::MinionStart
+  attr_reader :salt_event, :event_parsed_data
+
+  def initialize(salt_event)
+    @salt_event = salt_event
+    @event_parsed_data = @salt_event.parsed_data
```

Comment:
I don't want to memoize parsed_data for the reasons explained [here](https://github.com/SUSE/pharos/pull/5#discussion_r97799805). Then I though I would need that data many times and I created this instance variable but after all I didn't even use it. I think I should remove this variable as you suggest but not memoize the ```parsed_data``` method. If some code needs parsed data mutliple times I prefer to store it in a local var.

If both of you @mssola @ereslibre think we won't step into bugs I will memoize the method.